src/pocketdb/repositories/web/WebRpcRepository.cpp

```diff
@@ -2129,7 +2129,7 @@ namespace PocketDb
 
         string sql = R"sql(
             select t.Id
-            from Transactions t indexed by Transactions_Type_Last_String1_Height_Id
```

Comment:
I'm not sure about this index - String1 is not involved in the query and it stands in the index up to the Height